src/main/resources/static/test-studies.tsv

```diff
@@ -0,0 +1,5 @@
+study_A
+study_B
+study_C
+study_site_1
+study_site_2
```

Comment:
This red crossed circle with return is a sign that you forgot to leave empty line at the end of your file. You can configure Intellij to do it automatically.